pylearn2/datasets/svhn.py

```diff
@@ -29,14 +30,28 @@ class SVHN(dense_design_matrix.DenseDesignMatrixPyTables):
 
     Parameters
     ----------
-    which_set : WRITEME
-    path : WRITEME
-    center : WRITEME
-    scale : WRITEME
-    start : WRITEME
-    stop : WRITEME
-    axes : WRITEME
-    preprocessor : WRITEME
+    which_set : str
+                Subset name.
```

Comment:
There is no official valid. There original sets are: train, extra, and test.
train_all: train + extra
splitted_train: train + extra - valid